openverse_catalog/dags/common/requester.py

```diff
@@ -50,23 +49,24 @@ def __init__(self, delay=0, headers=None):
         self._last_request = 0
         self.session = requests.Session()
 
-    def get(self, url, params=None, **kwargs):
+    def _make_request(self, method, url, **kwargs):
         """
-        Make a get request, and return the response object if it exists.
+        Make a request, and return the response object if it exists.
 
         Required Arguments:
 
         url:      URL to make the request as a string.
         params:   Dictionary of query string params
-        **kwargs: Optional arguments that will be passed to `requests.get`
+        **kwargs: Optional arguments that will be passed to the `requests`
```

Comment:
The docstrings params need to be updated, too: add `method` and remove `params` (or keep it if it's possible to document a `**kwargs` parameter).

Reply:
Updated in https://github.com/WordPress/openverse-catalog/pull/865/commits/8c5d0a26a92313a93bbab4265eea2b11b0e1d26c.